File: tasks.py
class Tasks:
    def add(self, name, priority):
        # TODO

    def next(self):
        # TODO

if __name__ == "__main__":
    t = Tasks()
    t.add("siivous",10)
    t.add("ulkoilu",50)
    t.add("opiskelu",50)
    print(t.next()) # opiskelu
    t.add("treffit",100)
    print(t.next()) # treffit
    print(t.next()) # ulkoilu

File: Java_template/Tasks.java
public class Tasks {
    public void add(String name, int priority) {
        // TODO
    }

    public String next() {
        // TODO
    }

    public static void main(String[] args) {
        Tasks t = new Tasks();
        t.add("siivous",10);
        t.add("ulkoilu",50);
        t.add("opiskelu",50);
        System.out.println(t.next()); // opiskelu
        t.add("treffit",100);
        System.out.println(t.next()); // treffit
        System.out.println(t.next()); // ulkoilu
    }
}
